install_script.sh.template

```diff
@@ -590,6 +587,49 @@ function install_installer() {
     return "$exit_status"
 }
 
+function remove_existing_packages_for_fips_flavor() {
+  local sudo_cmd="$1"
+  local os="$2"
+  local distribution="$3"
+  local packages_to_uninstall="$4"
```

Comment:
are these args needed?

Reply:
good catch, removed in https://github.com/DataDog/agent-linux-install-script/pull/327/commits/45b0dd178ffb9075f5487a85ac37e7bebb96681f